File: EaterNet/Eater/Models/Customer.cs
namespace Eater.Models
{
    public class Customer
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Password { get; set; }
        public List<ProductOrder> ProductOrders { get; set; } = new();

        public Customer(string id, string name, string password)
        {
            Id = id;
            Name = name;
            Password = password;
        }
    }
}

File: EaterSpring/src/main/kotlin/xyz/purema/eater/models/Customer.kt
package xyz.purema.eater.models

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.OneToMany

@Entity
data class Customer(
    @Id
    val id: String,
    var name: String,
    var password: String
) {
    @OneToMany(mappedBy = "customer")
    var productOrders: List<ProductOrder> = emptyList()
}